kai-analyzer/main.go

```diff
@@ -29,18 +31,22 @@ func main() {
 		panic(fmt.Errorf("rules directory must be valid"))
 	}
 
-	file, err := os.Create("kai-analyzer.log")
-	if err != nil {
-		panic(err)
-	}
-	logger := slog.NewJSONHandler(file, &slog.HandlerOptions{
+	logger := slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{
```

Comment:
why not a file?

Reply:
because I can just stream stderr directly to the VSCode outputchannel

Reply:
Lots of other LSP's end up using a file, and because we will not just be using it from the LSP, but from KAI as well, what would be the donwside of using a file?

Reply:
I think, in the end that is fine, right now streaming it directly to the console of vscode has been incredibly convenient.

Reply:
does a combined handler work?